x-pack/plugin/ccr/src/internalClusterTest/java/org/elasticsearch/xpack/ccr/AutoFollowIT.java

```diff
@@ -118,7 +118,7 @@ public void testAutoFollow() throws Exception {
         createLeaderIndex("metrics-201901", leaderIndexSettings);
 
         createLeaderIndex("logs-201901", leaderIndexSettings);
-        assertLongBusy(() -> { assertTrue(ESIntegTestCase.indexExists("copy-logs-201901", followerClient())); });
+        ESIntegTestCase.awaitIndexExists("copy-logs-201901", followerClient(), TimeValue.timeValueSeconds(120));
```

Comment:
10s should be fine here unless something is _desperately_ broken. The auto-follow is triggered by long-polling the cluster state on the leader so it should react immediately to the index creation.

Reply:
OK makes sense. I dropped it.